src/types/payload.ts

```diff
@@ -16,6 +16,9 @@ export enum GithubEvent {
 
   // pull_request
   PULL_REQUEST_OPENED = "pull_request.opened",
+
+  // installation event
+  INSTALLATION_ADDED_EVENT = "installation_repositories.added",
```

Comment:
Check the following case:
1. Partner creates `repository 1`
2. Partner adds bot to the `repository 1`
3. Partner creates `repository 2`
4. Partner sets bot's repository access so that bot could use `repository 2`

Does the `installation_repositories.added` fired on step 4 ?

Reply:
Yea, its fired.. Any addition of new repository gets it fired.

Not organizational addition